### PR TITLE
Recipe for integrate tagria lib  with vraptor

### PR DESCRIPTION
**Pull request by jslsolucoes**
Cookbook learning how to integrate vraptor with tagrialib

### CODE REVIEW

**Comment by angeliski** [comment]
Hey @Turini And about that pull? Need something?

**Comment by Turini** [comment]
will take a look soon!

File: vraptor-site/content/en/cookbook/integrating-vraptor-with-tagrialib.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+title: Integrating VRaptor with Tagria
+---
+
+# Integrating VRaptor with Tagria
+
+## by Jonatan Lemes
+
+Tagria is an open source component suit framework that help build beautiful view in minutes. 
+Tagria integrates perfectly with any action based mvc framework for java including Vraptor.
```
Comment (Turini) [inline vraptor-site/content/en/cookbook/integrating-vraptor-with-tagrialib.html]: could you update Vraptor to VRaptor? all references, pease

File: vraptor-site/content/pt/cookbook/integrando-vraptor-com-tagrialib.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+title: Integrando  VRaptor com Tagria
+---
+
+# Integrando  VRaptor com Tagria
```
Comment (Turini) [inline vraptor-site/content/pt/cookbook/integrando-vraptor-com-tagrialib.html]: there is an extra space between "Integrando" and "VRaptor". Same on the title

File: vraptor-site/content/pt/cookbook/integrando-vraptor-com-tagrialib.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+## by Jonatan Lemes
```
Comment (Turini) [inline vraptor-site/content/pt/cookbook/integrando-vraptor-com-tagrialib.html]: [by -> por] Jonatan Lemes

**Comment by angeliski** [comment]
if you need, I can help change to approve the pull requests

**Comment by Turini** [comment]
hey @jslsolucoes, thank you for this new doc! sounds great

I just left a few comments. It'll be merged as soon as it's done

**Comment by Turini** [comment]
> if you need, I can help change to approve the pull requests

sounds great to me, @angeliski! any help are very welcome 

@jslsolucoes is it ok for you?

**Comment by jslsolucoes** [comment]
@Turini it`s ok for me and @angeliski appreciate your help and contribution

**Comment by angeliski** [comment]
@Turini and @jslsolucoes , I can't update that pull, because the fork was deleted, that issue is about that https://github.com/isaacs/github/issues/168. I can open another pull with these updates? Any problem?

**Comment by jslsolucoes** [comment]
@angeliski @Turini Really sorry, in fact i deleted the fork after doing this pull and forget about post modifications :( .  @angeliski no problem for me opening a new pull with content and modifications if you can. Sorry again guys.

**Comment by angeliski** [comment]
No problem @jslsolucoes, I will create that pull soon, thanks!

**Comment by jslsolucoes** [comment]
Closing pull request in favor of https://github.com/caelum/vraptor4/pull/1081

**Pull request closed without merging**